### ISSUE TITLE
Unable to add custom repositories

### ISSUE THREAD

**Issue by link6155** [hacs/integration#2157]
### Web browser

Edge

### Web browser version

Beta 93.0.961.27 ARM64

## System Health

version | core-2021.8.8
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.9.6
os_name | Linux
os_version | 5.10.17-v8
arch | aarch64
timezone | America/Denver

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
Github API Calls Remaining | 4745
Installed Version | 1.14.0
Stage | running
Available Repositories | 870
Installed Repositories | 12

</details>

<details><summary>AccuWeather</summary>

can_reach_server | ok
-- | --
remaining_requests | 26

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | September 13, 2021, 6:00 PM
relayer_connected | true
remote_enabled | true
remote_connected | true
alexa_enabled | false
google_enabled | true
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | pending

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 6.2
-- | --
update_channel | stable
supervisor_version | supervisor-2021.06.8
docker_version | 20.10.6
disk_total | 28.6 GB
disk_used | 7.6 GB
healthy | true
supported | true
board | rpi4-64
supervisor_api | ok
version_api | ok
installed_addons | File editor (5.3.3), Terminal & SSH (9.1.3), Visual Studio Code (3.6.2), Home Assistant Google Drive Backup (0.104.3), Node-RED (10.0.0), Samba share (9.5.1)

</details>

<details><summary>Lovelace</summary>

dashboards | 2
-- | --
resources | 8
views | 5
mode | storage

</details>

### Checklist

- [X] I'm running the newest version of HACS <https://github.com/hacs/integration/releases/latest>
- [X] I have filled out the issue template to the best of my ability.
- [X] I have read <https://hacs.xyz/docs/issues>
- [X] This issue is related to the frontend of HACS.
- [X] This issue only contains 1 issue (if you have multiple issues, open one issue for each issue).

### Describe the issue

Unable to add custom repositories in HACS frontend.

### Reproduction steps

1. Navigate to HACS > Integrations > Menu > Custom Repositories
2. Insert custom repository URL (e.g. https://github.com/DBuit/light-popup-card) 
3. Select category (e.g. Lovelace)
3. Press "Add"

### Screenshots

![image](https://user-images.githubusercontent.com/1360980/130490791-4e387f04-88c8-4710-97eb-0d3b1f8b7d1a.png)

### Javascript logs from your browser console

```text
app.4e133b2d.js:5232 The main 'lit-element' module entrypoint is deprecated. Please update your imports to use the 'lit' package: 'lit' and 'lit/decorators.ts' or import from 'lit-element/lit-element.ts'.
55704 @ app.4e133b2d.js:5232
s @ app.4e133b2d.js:30594
53918 @ app.4e133b2d.js:1258
s @ app.4e133b2d.js:30594
(anonymous) @ app.4e133b2d.js:30594
(anonymous) @ app.4e133b2d.js:36703
(anonymous) @ app.4e133b2d.js:36703
main-60575625.js:1217 The main 'lit-element' module entrypoint is deprecated. Please update your imports to use the 'lit' package: 'lit' and 'lit/decorators.ts' or import from 'lit-element/lit-element.ts'.
(anonymous) @ main-60575625.js:1217
integrations:1 [Intervention] Images loaded lazily and replaced with placeholders. Load events are deferred. See https://go.microsoft.com/fwlink/?linkid=2048113
c.75898fa6.js:57 Couldnt play ( fade-in-animation ). TypeError: document.timeline.play is not a function
    at HTMLElement._configureAnimations (c.75898fa6.js:57)
    at HTMLElement.playAnimation (c.75898fa6.js:57)
    at HTMLElement._renderOpened (c.75898fa6.js:57)
    at HTMLElement.__openedChanged (c.75898fa6.js:38)
    at HTMLElement.<anonymous> (c.75898fa6.js:38)
_configureAnimations @ c.75898fa6.js:57
playAnimation @ c.75898fa6.js:57
_renderOpened @ c.75898fa6.js:57
__openedChanged @ c.75898fa6.js:38
(anonymous) @ c.75898fa6.js:38
requestAnimationFrame (async)
__deraf @ c.75898fa6.js:38
_openedChanged @ c.75898fa6.js:38
_openedChanged @ c.75898fa6.js:57
Hn @ main-60575625.js:266
xn @ main-60575625.js:266
_propertiesChanged @ main-60575625.js:266
_flushProperties @ main-60575625.js:266
_invalidateProperties @ main-60575625.js:266
Object.defineProperty.set @ main-60575625.js:266
open @ c.75898fa6.js:38
open @ c.75898fa6.js:115
toggle @ c.75898fa6.js:115
(anonymous) @ main-60575625.js:266
Li @ main-60575625.js:266
Fi @ main-60575625.js:266
click @ main-60575625.js:266
Ei @ main-60575625.js:266
c.75898fa6.js:57 Couldnt play ( paper-menu-grow-width-animation ). TypeError: document.timeline.play is not a function
    at HTMLElement._configureAnimations (c.75898fa6.js:57)
    at HTMLElement.playAnimation (c.75898fa6.js:57)
    at HTMLElement._renderOpened (c.75898fa6.js:57)
    at HTMLElement.__openedChanged (c.75898fa6.js:38)
    at HTMLElement.<anonymous> (c.75898fa6.js:38)
_configureAnimations @ c.75898fa6.js:57
playAnimation @ c.75898fa6.js:57
_renderOpened @ c.75898fa6.js:57
__openedChanged @ c.75898fa6.js:38
(anonymous) @ c.75898fa6.js:38
requestAnimationFrame (async)
__deraf @ c.75898fa6.js:38
_openedChanged @ c.75898fa6.js:38
_openedChanged @ c.75898fa6.js:57
Hn @ main-60575625.js:266
xn @ main-60575625.js:266
_propertiesChanged @ main-60575625.js:266
_flushProperties @ main-60575625.js:266
_invalidateProperties @ main-60575625.js:266
Object.defineProperty.set @ main-60575625.js:266
open @ c.75898fa6.js:38
open @ c.75898fa6.js:115
toggle @ c.75898fa6.js:115
(anonymous) @ main-60575625.js:266
Li @ main-60575625.js:266
Fi @ main-60575625.js:266
click @ main-60575625.js:266
Ei @ main-60575625.js:266
c.75898fa6.js:57 Couldnt play ( paper-menu-grow-height-animation ). TypeError: document.timeline.play is not a function
    at HTMLElement._configureAnimations (c.75898fa6.js:57)
    at HTMLElement.playAnimation (c.75898fa6.js:57)
    at HTMLElement._renderOpened (c.75898fa6.js:57)
    at HTMLElement.__openedChanged (c.75898fa6.js:38)
    at HTMLElement.<anonymous> (c.75898fa6.js:38)
_configureAnimations @ c.75898fa6.js:57
playAnimation @ c.75898fa6.js:57
_renderOpened @ c.75898fa6.js:57
__openedChanged @ c.75898fa6.js:38
(anonymous) @ c.75898fa6.js:38
requestAnimationFrame (async)
__deraf @ c.75898fa6.js:38
_openedChanged @ c.75898fa6.js:38
_openedChanged @ c.75898fa6.js:57
Hn @ main-60575625.js:266
xn @ main-60575625.js:266
_propertiesChanged @ main-60575625.js:266
_flushProperties @ main-60575625.js:266
_invalidateProperties @ main-60575625.js:266
Object.defineProperty.set @ main-60575625.js:266
open @ c.75898fa6.js:38
open @ c.75898fa6.js:115
toggle @ c.75898fa6.js:115
(anonymous) @ main-60575625.js:266
Li @ main-60575625.js:266
Fi @ main-60575625.js:266
click @ main-60575625.js:266
Ei @ main-60575625.js:266
c.75898fa6.js:57 Couldnt play ( fade-out-animation ). TypeError: document.timeline.play is not a function
    at HTMLElement._configureAnimations (c.75898fa6.js:57)
    at HTMLElement.playAnimation (c.75898fa6.js:57)
    at HTMLElement._renderClosed (c.75898fa6.js:57)
    at HTMLElement.__openedChanged (c.75898fa6.js:38)
    at HTMLElement.<anonymous> (c.75898fa6.js:38)
_configureAnimations @ c.75898fa6.js:57
playAnimation @ c.75898fa6.js:57
_renderClosed @ c.75898fa6.js:57
__openedChanged @ c.75898fa6.js:38
(anonymous) @ c.75898fa6.js:38
requestAnimationFrame (async)
__deraf @ c.75898fa6.js:38
_openedChanged @ c.75898fa6.js:38
_openedChanged @ c.75898fa6.js:57
Hn @ main-60575625.js:266
xn @ main-60575625.js:266
_propertiesChanged @ main-60575625.js:266
_flushProperties @ main-60575625.js:266
_invalidateProperties @ main-60575625.js:266
Object.defineProperty.set @ main-60575625.js:266
close @ c.75898fa6.js:38
close @ c.75898fa6.js:115
_onIronActivate @ c.75898fa6.js:115
(anonymous) @ main-60575625.js:266
fire @ main-60575625.js:266
_itemActivate @ c.75898fa6.js:1
_activateHandler @ c.75898fa6.js:1
_activateHandler @ c.75898fa6.js:1
(anonymous) @ main-60575625.js:266
Li @ main-60575625.js:266
Fi @ main-60575625.js:266
click @ main-60575625.js:266
Ei @ main-60575625.js:266
c.75898fa6.js:57 Couldnt play ( paper-menu-shrink-width-animation ). TypeError: document.timeline.play is not a function
    at HTMLElement._configureAnimations (c.75898fa6.js:57)
    at HTMLElement.playAnimation (c.75898fa6.js:57)
    at HTMLElement._renderClosed (c.75898fa6.js:57)
    at HTMLElement.__openedChanged (c.75898fa6.js:38)
    at HTMLElement.<anonymous> (c.75898fa6.js:38)
_configureAnimations @ c.75898fa6.js:57
playAnimation @ c.75898fa6.js:57
_renderClosed @ c.75898fa6.js:57
__openedChanged @ c.75898fa6.js:38
(anonymous) @ c.75898fa6.js:38
requestAnimationFrame (async)
__deraf @ c.75898fa6.js:38
_openedChanged @ c.75898fa6.js:38
_openedChanged @ c.75898fa6.js:57
Hn @ main-60575625.js:266
xn @ main-60575625.js:266
_propertiesChanged @ main-60575625.js:266
_flushProperties @ main-60575625.js:266
_invalidateProperties @ main-60575625.js:266
Object.defineProperty.set @ main-60575625.js:266
close @ c.75898fa6.js:38
close @ c.75898fa6.js:115
_onIronActivate @ c.75898fa6.js:115
(anonymous) @ main-60575625.js:266
fire @ main-60575625.js:266
_itemActivate @ c.75898fa6.js:1
_activateHandler @ c.75898fa6.js:1
_activateHandler @ c.75898fa6.js:1
(anonymous) @ main-60575625.js:266
Li @ main-60575625.js:266
Fi @ main-60575625.js:266
click @ main-60575625.js:266
Ei @ main-60575625.js:266
c.75898fa6.js:57 Couldnt play ( paper-menu-shrink-height-animation ). TypeError: document.timeline.play is not a function
    at HTMLElement._configureAnimations (c.75898fa6.js:57)
    at HTMLElement.playAnimation (c.75898fa6.js:57)
    at HTMLElement._renderClosed (c.75898fa6.js:57)
    at HTMLElement.__openedChanged (c.75898fa6.js:38)
    at HTMLElement.<anonymous> (c.75898fa6.js:38)
```

**Comment by strehlma** [comment]
I have the same issue!

**Comment by mattwalden014** [comment]
Confirmed this bug was introduced in the release of 1.14.0. No issue in 1.13.2

**Comment by ludeeus** [comment]
That is correct, was introduced with <https://github.com/hacs/integration/pull/2144> and fixed with #2158 
It only applies to non-lowercase owner/repository repositories
